packages/camera/camera_windows/windows/camera.cpp

```diff
@@ -16,6 +16,24 @@ constexpr char kVideoRecordedEvent[] = "video_recorded";
 constexpr char kCameraClosingEvent[] = "camera_closing";
 constexpr char kErrorEvent[] = "error";
 
+// Camera error codes
+constexpr char kCameraAccessDenied[] = "CameraAccessDenied";
+constexpr char kCameraError[] = "camera_error";
+constexpr char kPluginDisposed[] = "plugin_disposed";
+
+std::string GetErrorCode(CameraResult result) {
+  assert(result != CameraResult::kSuccess);
+
+  switch (result) {
+    case CameraResult::kAccessDenied:
+      return kCameraAccessDenied;
+
+    default:
+    case CameraResult::kError:
+      return kCameraError;
```

Comment:
Replace the default with a fallthrough with both remaining cases listed.

We should probably add a lint that bans `default` cases like we do elsewhere, so that we're forced to deal with each usage site if new enum values are ever added.

Reply:
I added all the possible cases, however, I wasn't able to remove the `default` case as that resulted in this error:

```
... error C2220: the following warning is treated as an error ...
... warning C4715: 'camera_windows::GetErrorCode': not all control paths return a value ...
camera_windows/example unit tests failed to build.
```

I moved the `default` case to be the fall-through after all other cases.